### PR TITLE
fix(library/nexus-schema): drop graphql-yoga for express

### PR DESCRIPTION
**Pull request by jasonkuhrt**

### CODE REVIEW

File: libraries/nexus-schema/tsconfig.json
Comment (jasonkuhrt) [inline libraries/nexus-schema/tsconfig.json:3]: for express

File: libraries/nexus-schema/src/server.ts
Comment (janpio) [inline libraries/nexus-schema/src/server.ts]:
```suggestion
    `🚀 Server ready at: http://localhost:4000/graphql`,
```

**Comment by jasonkuhrt** [comment]
Able to reproduce locally https://github.com/prisma/e2e-tests/pull/599/checks?check_run_id=902653053#step:4:107

Don't understand why it is happening.

Able to reproduce locally https://github.com/prisma/e2e-tests/pull/599/checks?check_run_id=902653053#step:4:115

Don't understand why it is happening.

**Comment by jasonkuhrt** [comment]
@janpio should be ready to go now

File: libraries/nexus-schema/src/server.ts
Comment (janpio) [inline libraries/nexus-schema/src/server.ts:12]: Any idea why this is needed @jasonkuhrt? The examples on https://github.com/graphql/express-graphql#simple-setup don't use it.

**Review by janpio — APPROVED** [review]

File: libraries/nexus-schema/src/server.ts
Comment (jasonkuhrt) [reply libraries/nexus-schema/src/server.ts:12]: That's the way resolver context gets created for every request. If a developer does not provide it, then express-graphql defaults to making the resolver context be the request object.

Comment (jasonkuhrt) [reply libraries/nexus-schema/src/server.ts:12]: https://github.com/graphql/express-graphql/issues/187

**Comment by jasonkuhrt** [comment]
@janpio is the policy here for devs to merge their own PRs after approval?

File: libraries/nexus-schema/src/server.ts
Comment (janpio) [reply libraries/nexus-schema/src/server.ts:12]: Why does the default not work for our project?

**Comment by janpio** [comment]
We do not have clear guideline for this - so I will just merge this when the test have run.

File: libraries/nexus-schema/src/server.ts
Comment (jasonkuhrt) [reply libraries/nexus-schema/src/server.ts:12]: Because we need to expose prisma client on the resolver context.

I don't think we need a context function though. That's just how it was setup before.

**Pull request merged**
